app/static/js/components/Experiments/Experiments.jsx

```diff
@@ -1,85 +1,78 @@
-import InputLabel from '@material-ui/core/InputLabel';
-import MenuItem from '@material-ui/core/MenuItem';
-import FormControl from '@material-ui/core/FormControl';
-import Select from '@material-ui/core/Select';
-import IngredientFrequency from './IngredientFrequency/IngredientFrequencyBarChart'
-import WordCloud from './WordCloud/WordCloud'
+import NavBar from "components/core/NavBar/NavBar";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import IngredientFrequency from "./IngredientFrequency/IngredientFrequencyBarChart";
+import WordCloud from "./WordCloud/WordCloud";
+import styles from "./styles";
 
 class Experiments extends React.Component {
-
     constructor(props) {
-      super(props);
-      this.state = {
-        experiments: 'ingredient_frequency',
-      };
+        super(props);
+        this.state = {
+            experiments: "ingredient_frequency"
+        };
     }
 
     renderExperiment() {
-      const { experiments } = this.state
-      if (experiments === 'word_cloud') {
-        return <WordCloud maxWidth={600} maxHeight={500} />
-      }
-      return <IngredientFrequency maxWidth={600} maxHeight={500} />
+        const { experiments } = this.state;
+        if (experiments === "word_cloud") {
+            return <WordCloud maxWidth={600} maxHeight={500} />;
+        }
+        return <IngredientFrequency maxWidth={600} maxHeight={500} />;
     }
 
-    render () {
-      const { classes } = this.props;
-      return (
-        <NavBar>
-          <div>
-            <form className={classes.root} autoComplete="off">
-              <FormControl className={classes.formControl}>
-                <InputLabel htmlFor="experiments">Experiments</InputLabel>
-                <Select
-                  value={this.state.experiments }
-                  onChange={event => {
-                    this.setState({ [event.target.name]: event.target.value });
-                  }}
-                  inputProps={{
-                    name: 'experiments',
-                    id: 'experiments',
-                  }}>
-                  {/* <MenuItem value={"radial_network_graph"}>Radial Network Graph</MenuItem>
+    render() {
+        const { classes } = this.props;
+        return (
+            <NavBar>
+                <div>
+                    <form className={classes.root} autoComplete="off">
+                        <FormControl className={classes.formControl}>
+                            <InputLabel htmlFor="experiments">
+                                Experiments
+                            </InputLabel>
+                            <Select
+                                value={this.state.experiments}
+                                onChange={event => {
```

Comment:
```suggestion
                                onChange={event => {
                                    this.setState({
                                        [event.target.name]: event.target.value
                                    });
                                }}
```
Let's move this to an actual function so that in the case of a re-render it does rebind.